ipfsApi/client.py

```diff
@@ -1793,12 +1792,37 @@ def files_mv(self, source, dest, **kwargs):
     ###########
 
     @utils.return_field('Hash')
+    def add_bytes(self, data, **kwargs):
```

Comment:
Good idea. I hadn't thought to handle the different kinds of string in Python separately (b'' vs u''), but it makes sense.